packages/core/src/auth-client.ts

```diff
@@ -42,9 +43,15 @@ export class AuthClient {
    */
   static getInstance(authClientConfig?: UIAuthConfig, store?: Store, cryptoUtils?: CryptoUtils): UIAuthClient {
     if (!AuthClient.instance) {
+      const DEFAULT_NAME: string = 'carbon.super';
+
       const extendedAuthClientConfig: UIAuthConfig = {
         ...authClientConfig,
+        enableConsoleBranding: authClientConfig?.enableConsoleBranding ?? true,
+        enableConsoleTextBranding: authClientConfig?.enableConsoleTextBranding ?? true,
```

Comment:
Do we need the `console` part here? How about `enableBranding` and `enableTextBranding`? @brionmario @dasuni-30 WDYT?

Reply:
Branding is taken from props as well. 
Need to specify this is coming from asgardeo/IS, right?